eZ/Publish/Core/MVC/Symfony/Routing/UrlAliasRouter.php

```diff
@@ -315,7 +315,7 @@ public function generate($name, $parameters = [], $referenceType = UrlGeneratorI
                 // Check if location is a valid Location object
                 if (isset($parameters['location']) && !$parameters['location'] instanceof Location) {
                     throw new LogicException(
-                        "When generating an UrlAlias route, 'location' parameter must be a valid eZ\\Publish\\API\\Repository\\Values\\Content\\Location."
+                        "When generating a UrlAlias route, the 'location' parameter must be a valid eZ\\Publish\\API\\Repository\\Values\\Content\\Location."
```

Comment:
Let's go pro here:
```suggestion
                        "When generating a UrlAlias route, the 'location' parameter must be a valid " . Location::class . '.'
```
(no strong feelings for the trailing dot here)

Reply:
We can go even deeper:
```suggestion
sprintf("When generating a UrlAlias route, the 'location' parameter must be a valid %s.", Location::class)
```